spec/features/preassembly_gis_accessioning_spec.rb

```diff
@@ -149,6 +149,21 @@
     expect(page).to have_text('Scale not given ; EPSG::3310') # map data
     expect(page).to have_text('Geospatial data') # genre
     expect(page).to have_text('Cartographic dataset') # genre
-    expect(page).to have_link('View in EarthWorks') # link to Earthworks
```

Comment:
this check is duplicative of the one above that already verifies the full link with URL